tests/cases/fourslash/incompleteFunctionCallCodefix3.ts

```diff
@@ -3,4 +3,6 @@
 // @noImplicitAny: true
 //// function ...q) {}} f(10);
 
-verify.not.codeFixAvailable();
+verify.not.codeFixAvailable([

```

Comment:
Forgot to remove `.not`

Reply:
I do believe this test to check for the the specific codefix not being generated. However the test file contains a syntax error, which leads to a codefix being generated by this change. In order for this test to successfully run I assume that either the test fixture must be changed or the test expectation be widened to check for only the specific codefix not being generated.

Reply:
It's better to avoid `verify.not.codeFixAvailable`, since such a test could easily pass if the code fix does exist but has a slightly different description. Better to use `verify.codeFixAvailable` and list exactly the codefixes that exist.

Reply:
Since the new codefix is now only being generated inside `for ... in/of` loops I was able to revert the changes to this file completely.